apps/coscrad-frontend/src/components/resources/bibliographic-references/bibliographic-reference-index.presenter.tsx

```diff
@@ -0,0 +1,13 @@
+import { BibliographicReferenceIndexState } from '../../../store/slices/resources';
+import { GenericIndexPresenter } from '../../../utils/generic-components/presenters/generic-index-presenter';
+
+export const BibliographicReferenceIndexPresenter = (
+    indexResult: BibliographicReferenceIndexState
+) => (
+    /**
+     *  TODO [https://www.pivotaltracker.com/story/show/183681839]
+     * We may some day read the actions and allow for bulk command execution in
+     * an index view.
+     */
+    <GenericIndexPresenter {...indexResult} />
```

Comment:
Currently, there is no resource type on the view model itself. That's because you get the resource type from location (i.e. the route). Think about it- you have to know the resource type to request the data from Redux in the first place.

I'd prefer to keep it like this. Let me know if it proves too difficult to work around.